doc/modules/clustering.rst

```diff
@@ -137,8 +137,9 @@ mixture models. KMeans can be seen as a special case of Gaussian mixture
 model with equal covariance per component.
 
 :term:`Transductive <transductive>` clustering methods (in contrast to
-:term:`inductive` clustering methods) are not designed to be applied to new,
-unseen data.
```

Comment:
This marks the key difference between Transductive and Inductive Clustering. Therefore, I think it should be preseverd.